server/api.js

```diff
@@ -212,57 +196,47 @@ router.post("/register", validInfo, async (req, res) => {
     lastName,
     userRole,
     userEmail,
+    github_id,
```

Comment:
🔴 We shouldn't get the `github_id` from the `req.body` here, the user can camper with the request body. Instead we should use the `req.session.githubId`, which we previously set in '/githubAuth', line 262.

Reply:
@petemcfarlane what do mean exactly by 'the user can camper with the request body'?

Reply:
I think we sort it out, can you have a look at it please?

Reply:
sorry I meant "tamper" not "camper", as in the user can change the request body, but they can't change the session, that's controlled by us in the API.